linode/manager
# test: [M3-8393] - Cypress test for Account Maintenance CSV downloads

## Description 📝
Added cypress test for validating downloaded csv content of account maintenance with Pending and Completed status 

## Changes  🔄
Added below cypress tests under in spec file - cypress/e2e/core/account/account-maintenance.spec.ts
- confirm maintenance details in the tables


## Jira
- M3-8393

## How to test 🧪
`yarn cy:run -s cypress/e2e/core/account/account-maintenance.spec.ts`

### Verification steps
When test executed using this command  `yarn cy:run -s cypress/e2e/core/account/account-maintenance.spec.ts`all tests should pass successfully and you should see new test added as per below screenshot 

![image](https://github.com/user-attachments/assets/fb3377f8-4108-4230-9693-55206fb7dc40)


## As an Author I have considered 🤔

*Check all that apply*
